core/storage/base_model/gae_models.py

```diff
@@ -109,17 +115,29 @@ def get_multi(cls, entity_ids, include_deleted=False):
 
     @classmethod
     def put_multi(cls, entities):
+        """Stores sequence of entities.
+
+        Args:
+            entities: sequence.
+
+        Return:
+            list with the stored keys.
+        """
         return ndb.put_multi(entities)
 
     def delete(self):
         super(BaseModel, self).key.delete()
 
     @classmethod
     def get_all(cls, include_deleted_entities=False):
```

Comment:
_include_deleted_entities_ here and _include_deleted_ on line 95 probably should have same name. I think include_deleted is better.